### ISSUE TITLE
missing write file descriptor

### ISSUE THREAD

**Issue by makkalot** [pdfcpu/pdfcpu#217]
Hi, 

when using the project inside of go.mod, by default go will fetch `0.3.4` version of the lib. When I try to apply watermark with in memory writer with a code like this : 

```
var b bytes.Buffer
outWriter := bufio.NewWriter(&b)

// this means it's a watermark
onTop := false
wm, err := pdfcpu.ParseTextWatermarkDetails(text, "points:48, scale:1, op:.4", onTop)
if err != nil {
	return nil, err
}

err = api.AddWatermarks(readSeek, outWriter, nil, wm, nil)
if err != nil {
	return nil, fmt.Errorf("applying watermark failed : %w", err)
}
```

I'd get the error above. When I point my go.mod to master, the issue is gone. Please release 0.3.5 :)

**Comment by hhrutter** [comment]
Hello!

Yes, I am aware of this!
Like the ReadMe  says I highly recommend working with the latest commit.

Watch out for the next release within the next couple of days!
Thank you for using pdfcpu 💚

**Comment by hhrutter** [comment]
Fixed with [v0.3.5](https://github.com/pdfcpu/pdfcpu/releases/tag/v0.3.5)
